vscode/package.json

```diff
@@ -42,5 +43,10 @@
   "scripts": {
     "tsc:check": "node ../node_modules/typescript/bin/tsc -p ./tsconfig.json",
     "build": "npm run tsc:check && node build.js"
-  }
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/qsharp.git"
+  },
+  "license": "SEE LICENSE IN LICENSE.txt"
```

Comment:
Is this a common pattern? I've not seen this before. Being that the license is just standard MIT, I believe this should just be 'MIT'.

Reply:
Yes, this is a pattern specified [here](https://code.visualstudio.com/api/references/extension-manifest), search for "SEE LICENSE".